plugins/tag_search/lib/model/index/TagIndex.php

```diff
@@ -43,7 +43,7 @@ public static function getIndexFieldsMap()
 		{
 			self::$fieldsMap = array(
 				'int_id' => 'intId',
-				'tag' => 'TagWithEqual',
+				'tag' => 'tag',
```

Comment:
I don't see how I can align this using IndexSchema.xml
@hilak, do you know why it was implemented like that?